wfe/wfe.go

```diff
@@ -983,8 +984,11 @@ func (wfe *WebFrontEndImpl) Challenge(
 
 	authz, err := wfe.SA.GetAuthorization(ctx, authorizationID)
 	if err != nil {
-		// TODO(#1198): handle db errors etc
-		notFound()
+		if err == sql.ErrNoRows {
+			notFound()
+		} else {
+			wfe.sendError(response, logEvent, probs.ServerInternal("Internal Server Error"), err)
```

Comment:
Following the example in `Authorization()`, this should say something like "unable to find authorization". We don't generally put the words "Internal Server Error" inside a `probs.ServerInternal`, since it's redundant.